_tests_/PUT_channel.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { generateRandomString } from '../_helpers_/common'
+import { createChannel, updateChannel, deleteChannel } from '../_helpers_/api.helper'
+
+describe('PUT /api/channel/:id/update', function () {
+    var channel_id: number[] = [];
+
+    test('CHA-001 Verify that user can change total details of an existent channel', async function () {
+        const randomName = generateRandomString();
+        const randomDescription = generateRandomString();
+        const randomName_2 = generateRandomString();
+        const randomDescription_2 = generateRandomString()
+        // create new channel
+        const createdResponse = await createChannel(randomName, randomDescription, "#000000", false);
+        // console.log(createdResponse.body);
+        channel_id.push(createdResponse.body.id);
+        // update channel created with id
+        const updatedResponse = await updateChannel(createdResponse.body.id, randomName_2, randomDescription_2, "#FFFFFF", true);
+        // console.log(updatedResponse.body)
+        expect(updatedResponse.status).toEqual(200);
+        expect(updatedResponse.body.error).toEqual(0);
+        expect(updatedResponse.body.name.toLowerCase()).toEqual(randomName_2);
+        expect(updatedResponse.body.description).toEqual(randomDescription_2);
+        expect(updatedResponse.body.color).toEqual("#FFFFFF");
+        expect(updatedResponse.body.starred).toEqual(true);
+        expect(updatedResponse.body).toHaveProperty('id');
+    });
+
+    test('CHA-012 Verify that user cannot change details of an existent channel with existent "name" in Body', async function () {
```

Comment:
Not clear test summary. It should Can not rename channel to existing name.

Reply:
fixed